WordPress/WordPressTest/PostCardCellTests.swift

```diff
@@ -490,13 +490,22 @@ class PostCardCellTests: XCTestCase {
         expect(self.postCell.statusLabel.textColor).to(equal(UIColor.warning))
     }
 
-    func testShowsUnsavedChangesMessageWhenPostHasAutosave() {
+    func testShowsVersionConflictMessageWhenPostHasRemoteAutosave() {
         let post = PostBuilder(context).with(remoteStatus: .sync).autosaved().build()
 
         postCell.configure(with: post)
 
-        expect(self.postCell.statusLabel.text).to(equal(i18n("You've made unsaved changes to this post")))
-        expect(self.postCell.statusLabel.textColor).to(equal(UIColor.warning(.shade40)))
+        expect(self.postCell.statusLabel.text).to(equal(i18n("Version Conflict")))
```

Comment:
I understand the test below showing a version conflict as the post is constructed with `hasVersionConflict()`.  This one though isn't... and I'm failing to understand the logic.

Can you explain a bit why the version conflict is correct in this test?